### ISSUE TITLE
Remove references to obsolete browsers

### ISSUE THREAD

**Issue by AutomatedTester** [SeleniumHQ/selenium#8317]
## 🚀 Feature Proposal

Remove all references to obsolete browsers

## Motivation

Currently, there are large areas of the codebase that are not being used. We don't need that dead code anymore.

## Example

Such as `IEHTA` and `Konqueror` in [BrowserType.java](https://github.com/SeleniumHQ/selenium/blob/master/java/client/src/org/openqa/selenium/remote/BrowserType.java#L40 ). We also need to remove where those are being used in the code.

**Comment by bongosway** [comment]
@AutomatedTester 
for e.g. FIREFOX_PROXY is not used anywhere in the codebase, does `not used` == `obsolete`?
if yes, then I can remove all the `not used` code. 

regards.

**Comment by AutomatedTester** [comment]
let's start a PR with that and see where we go

**Comment by bongosway** [comment]
I will pass on this one, as this is a really good one for someone newer :-) than me to pick up.

**Comment by rookieInTraining** [comment]
@AutomatedTester if the work hasn't started on this yet. I would like to contribute to this.

**Comment by AutomatedTester** [comment]
> @AutomatedTester if the work hasn't started on this yet. I would like to contribute to this.

Go for it

**Comment by rookieInTraining** [comment]
@AutomatedTester 
Removed :
```
String FIREFOX_PROXY = "firefoxproxy";
String FIREFOX_CHROME = "firefoxchrome";
String GOOGLECHROME = "googlechrome";
String IEXPLORE_PROXY= "iexploreproxy";
String SAFARI_PROXY = "safariproxy";
String KONQUEROR = "konqueror";
String MOCK = "mock";
String IE_HTA="iehta";
```

Have kept `PhantomJS` as I'm not sure whether to remove it or not according to [this](https://github.com/ariya/phantomjs/issues/15344). Also have kept`Android, Iphone and IPad` even though there aren't any usages for them.

**Comment by rookieInTraining** [comment]
My bad, there are no usages or classes for PhatomJS present now in the code. Removed it.

**Comment by barancev** [comment]
Fixed by commit 67dd6aca81a27453c6d69e763c66c828d3e87be0
